GVFS/GVFS/CommandLine/StatisticsVerb.cs

```diff
@@ -0,0 +1,209 @@
+﻿using CommandLine;
+using GVFS.Common;
+using GVFS.Common.Database;
+using GVFS.Common.FileSystem;
+using GVFS.Common.Git;
+using GVFS.Common.NamedPipes;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace GVFS.CommandLine
+{
+    [Verb(StatisticsVerb.StatisticsVerbName, HelpText = "Get statistics for the health state of the repository")]
+    public class StatisticsVerb : GVFSVerb.ForExistingEnlistment

```

Comment:
Are there any functional tests that can be added for this new verb?

Reply:
Probably. I plan to discuss this with Jameson since I'm not especially familiar with functional tests so hopefully we can work on that in the coming days 👍